web/template/pro-hub/pro-deputy-details.gotmpl

```diff
@@ -13,6 +13,14 @@
         </h1>
         <div class="moj-button-menu">
             <div class="moj-button-menu__wrapper">
+                <a
+                    href="{{ sirius (printf "/api/v1/finance/reports/deputies/%d/download-billing-schedule?OPG-Bypass-Membrane=1" .DeputyDetails.ID) }}"
+                    role="button"
+                    draggable="false"
+                    class="govuk-button moj-button-menu__item govuk-button--secondary {{if eq .IsFinanceManager false}} hide{{end}}"
```

Comment:
Is it intentional that we hide the button with CSS instead of wrapping the <a> tag in a conditional?

Reply:
We tried wrapping the <a> and the logic did not work, but this way it does what we need it to do